patterns/guaranteed_receiver_nack.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"time"
+
+	"solace.dev/go/messaging"
+	"solace.dev/go/messaging/pkg/solace/config"
+	"solace.dev/go/messaging/pkg/solace/message"
+	"solace.dev/go/messaging/pkg/solace/resource"
+)
+
+func getEnv(key, def string) string {
+	if val, ok := os.LookupEnv(key); ok {
+		return val
+	}
+	return def
+}
+
+func main() {
+	// logging.SetLogLevel(logging.LogLevelInfo)
```

Comment:
@oodigie  Is it conventional to leave commented logging statements in the samples?

Reply:
@TrentDaniel,
Yes, that is the normal convention in the other samples written by the DAs.